go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/buildkite/yaml v0.0.0-20181016232759-0caa5f0796e3
 	github.com/coreos/go-semver v0.3.0
 	github.com/gin-gonic/gin v1.7.7
-	github.com/go-vela/server v0.13.0
-	github.com/go-vela/types v0.13.0
+	github.com/go-vela/server v0.13.2-0.20220426180142-90727d2b465a
```

Comment:
interesting. where is it getting v0.13.2 from?

Reply:
I believe that comes from the Go module system when pulling a dependency based on a commit.

Go sees the `v0.13.1` release is out there, but I'm vendoring from a commit published after that release.

So the system automatically updates to the next available patch release (`v0.13.2`).

What follows after that patch is the time and date (`20220426180142`).

And lastly, the shortened commit SHA that was vendored in (`90727d2b465a`).